README.md

```diff
@@ -273,7 +273,6 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 * [Syntax-Directed Variational Autoencoder for Structured Data](https://mlatgt.blog/2018/02/08/syntax-directed-variational-autoencoder-for-structured-data/)
 * [Weighted MinHash on GPU helps to find duplicate GitHub repositories.](https://blog.sourced.tech//post/minhashcuda/)
 * [Source Code Identifier Embeddings](https://blog.sourced.tech/post/id2vec/)
-* [Using recurrent neural networks to predict next tokens in the java solutions](http://near.ai/articles/2017-06-01-Code-Completion-Demo/)
```

Comment:
Thank you! Could you put https://codeforces.com/blog/entry/52327 instead of a broken link?

Reply:
cc @thecodingaviator ! Are you open to adding this new link ?

Reply:
@vcoisne Doing

Reply:
@EgorBu Done